app/main/views/briefs.py

```diff
@@ -102,6 +104,47 @@ def update_brief(brief_id):
     return jsonify(briefs=brief.serialize()), 200
 
 
+@main.route('/briefs/<int:brief_id>/admin', methods=['POST'])
```

Comment:
why put this in a separate endpoint?

Reply:
different behaviour, only certain fields allowed but editing is allowed after publishing